### DISCUSSION TITLE
how to make my own theme

### DISCUSSION THREAD

**Discussion by 7368697661** [namazso/SecureUxTheme#230] [General]
hey! love the tool, but do you know how i can make my own theme? all the themes on deviantart are paid these days, and the russian site you linked just puts me into a tab loop when i try to download. all i want is a simple amoled theme, so i was thinking i could make my own but i cant find any information or resources on the topic. do you have any ideas?

**Comment by lamteteeow** [comment]
@7368697661 Maybe you could checkout this guy's work to get some inspiration or reach out to him for instruction: 
- https://github.com/niivu
- https://www.deviantart.com/niivu
I am currently using gruvbox theme from him and it is okay-ish 👯

**Comment by rom4ster** [comment]
To create your own theme properly for windows is very difficult. The first thing you need to do is aquire vista style builder. Then you need to start messing around in there to create your msstyles. Most paid themes however have shell mods which are harder to look at. Overall doing all this will cost you money and time, or you can just pay cleodesktop for a good theme and hell support it basically 4ever and it costs like 10 bucks. 

If you go the first route, mydigitallife and virtualcustoms will immensly help you.

**Comment by 7368697661** [comment]
ah i see. i wish it was as easy as modding linux haha. do you know the best way to get in touch w cleo for a custom theme?

**Reply by rom4ster** [reply]
I just buy his themes and mod them
